tools/serverpod_cli/lib/src/language_server/language_server.dart

```
@@ -48,10 +43,7 @@ Future<void> runLanguageServer() async {
   });
 
   connection.onInitialized((_) async {
-    if (serverProject == null &&
-        exception is ServerpodModulesNotFoundException) {
-      _sendModulesNotFoundNotification(connection);
-    } else if (serverProject == null) {
```

Question: Are you sure this wasn't used? This will give you a notification in the right left corner if the loading of the Serverpod project failed. I'm sure we gave an error if we couldn't locate the modules, I believe we introduced that in the CLI to make sure the code generator didn't silently ignore to generate auth endpoints for example.

---

The only place in the code where this was thrown was the place that was safely removed. I think this error message has not been seen for a while.

---

Reverted the removal of the handling of this in the LSP.

In an upcoming PR I will reintrodue this throw once we now that we are only inspecting modules we are depending on.